packages/viz/src/utils/bloodglucose.js

```diff
@@ -25,46 +25,6 @@ import i18next from 'i18next'
 
 const t = i18next.t.bind(i18next)
 
-/**
- * classifyBgValue
- * @param {Object} bgBounds - object describing boundaries for blood glucose categories
- * @param {number} bgValue - integer or float blood glucose value in either mg/dL or mmol/L
- * @param {"threeWay" | "fiveWay"} classificationType - 'threeWay' or 'fiveWay'
- *
- * @return {String} bgClassification - low, target, high
- */
-export function classifyBgValue(bgBounds, bgValue, classificationType = 'threeWay') {
```

Comment:
You forgot to delete the tests. But that's not a big deal, I will delete them in my PR anyway :wink:

Reply:
(So as discussed yesterday, you have to delete them because I am not deleting them in my PR)

Reply:
So I don't have to delete it here ?

Reply:
You have to, since I ended up not deleting them

Reply:
Done